src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -4,22 +4,46 @@
 
 .hmrc-list-with-actions {
   @include govuk-font($size: 19);
+
+  .govuk-\!-font-weight-regular {
+    font-weight: normal !important;
+  }
+
+  @include govuk-media-query($from: desktop) {
+    .govuk-\!-summary-list-key-width {
+      width: 50% !important;
+    }
+
+    .govuk-\!-summary-list-actions-width {
+      width: 50% !important;
+    }
+  }
 }
 
-// Kept for backward compatibility with add-to-a-list
-.hmrc-list-with-actions--short {
+.hmrc-list-with-actions--long {
   @include govuk-media-query($from: desktop) {
+    .govuk-\!-summary-list-key-width {
+      width: 30% !important;
+    }
+
+    .govuk-\!-summary-list-actions-width {
+      width: 70% !important;
+    }
```

Comment:
There aren't equivalent classes for this – the closest ones are `govuk-!-width-two-thirds`, `govuk-!-width-one-third`. GOV.UK look like they'd suggest adding custom override classes here (https://design-system.service.gov.uk/get-started/extending-and-modifying-components/#custom-override-classes). In our case I think we'd want to add `.hmrc-\!-summary-list-key-width` & `.hmrc-\!-summary-list-actions-width`

Reply:
I have instead added `hmrc-summary-list__key` and `hmrc-summary-list__actions` because they don't override anything and they are new CSS keys